src/Core/Components/CollapsibleRegion/FluentCollapsibleRegion.razor.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Fast.Components.FluentUI.Utilities;
+using Microsoft.JSInterop;
+
+
+namespace Microsoft.Fast.Components.FluentUI;
+public partial class FluentCollapsibleRegion : FluentComponentBase
+{
+
+    internal enum CollapseState
+    {
+        Entering, Entered, Exiting, Exited
+    }
+
+    //internal double _height;
+    private bool _expanded, _isRendered;  //, _updateHeight;
+    //private ElementReference _wrapper;
+    //internal CollapseState _state = CollapseState.Exited;
+
+    protected string? StyleValue =>
+        new StyleBuilder()
```

Comment:
You always need to create a style starting by the dev style:  `new StyleBuilder(Style)...`